include/openthread/instance.h

```diff
@@ -102,6 +102,17 @@ otInstance *otInstanceInit(void *aInstanceBuffer, size_t *aInstanceBufferSize);
  */
 otInstance *otInstanceInitSingle(void);
 
+/**
+ * This method returns the identifer of the instance.
```

Comment:
Since we're reusing this for the CLI command documentation, can you update to not refer to a function/method.  That way it makes sense in both locations (API & CLI references).

`Gets the instance identifier.`

Reply:
Updated